### ISSUE TITLE
Build issue on Travis CI OSX with Xcode 9.4.1 and clang 9.1.0

### ISSUE THREAD

**Issue by tetrachrome** [google/googletest#3000]
Hi,

I recently updated my googletest submodule from version `1.7.0` to `1.10.0`, and wasn't able to get it build on Travis CI OSX (it was building fine with v1.7.0).

The failed job is at:
https://travis-ci.org/github/tetrachrome/Snowlake/jobs/722975146

```
Apple LLVM version 9.1.0 (clang-902.0.39.2)
72Target: x86_64-apple-darwin17.7.0
73Thread model: posix
74InstalledDir: /Applications/Xcode-9.4.1.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin
```

```
[ 25%] Building CXX object CMakeFiles/gtest.dir/src/gtest-all.cc.o
145/Applications/Xcode-9.4.1.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/clang++   -I/Users/travis/build/tetrachrome/Snowlake/libs/googletest/googletest/include -I/Users/travis/build/tetrachrome/Snowlake/libs/googletest/googletest  -Wall -Wshadow -Werror -Wconversion -DGTEST_HAS_PTHREAD=1 -fexceptions -W -Wpointer-arith -Wreturn-type -Wcast-qual -Wwrite-strings -Wswitch -Wunused-parameter -Wcast-align -Wchar-subscripts -Winline -Wredundant-decls -o CMakeFiles/gtest.dir/src/gtest-all.cc.o -c /Users/travis/build/tetrachrome/Snowlake/libs/googletest/googletest/src/gtest-all.cc
146In file included from /Users/travis/build/tetrachrome/Snowlake/libs/googletest/googletest/src/gtest-all.cc:38:
147In file included from /Users/travis/build/tetrachrome/Snowlake/libs/googletest/googletest/include/gtest/gtest.h:62:
148In file included from /Users/travis/build/tetrachrome/Snowlake/libs/googletest/googletest/include/gtest/internal/gtest-internal.h:40:
149/Users/travis/build/tetrachrome/Snowlake/libs/googletest/googletest/include/gtest/internal/gtest-port.h:836:12: error: no member named 'make_tuple' in namespace 'std'
150using std::make_tuple;
```

Basically I don't see `-std=c++11` or anything similar passed to clang, and looks like the compiler is operating under pre C++11 mode.

I'm not doing anything fancy with the build script, just something like:

```
mkdir build_dir && cd build_dir && cmake {GTEST_DIR} && VERBOSE=1 make
```

I can't seem to figure out how to get past this, and was wondering if any folks would give me a pointer.

Thanks in advance!

**Comment by Abidulislamalvy** [comment]
What

**Comment by derekmauro** [comment]
I believe this is fixed at head, but if you using 1.10.0, try passing `-DCMAKE_CXX_STANDARD=11` (or 14 or 17) to your `cmake` command. Let us know if this works for you.

**Comment by tetrachrome** [comment]
@derekmauro Setting `-DCMAKE_CXX_STANDARD=11` indeed fixes the issue. Thanks a ton!
